readme.md

```diff
@@ -21,12 +21,12 @@ Follow
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://<osherstiak>.github.io/layout_stars/)
+- [TEST REPORT LINK](https://<osherstiak>.github.io/layout_stars/report/html_report/)
```

Comment:
```suggestion
- [DEMO LINK](https://osherstiak.github.io/layout_stars/)
- [TEST REPORT LINK](https://osherstiak.github.io/layout_stars/report/html_report/)
```
fix your links

Reply:
All are fixed. Do I need to creat another pull request?